src/components/Nav/Admin/Admin.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { IoMdClose } from 'react-icons/io';
+
+// COMPONENTS
+import UserFormInputs from '../UserInput/UserInput';
+import UserFormBtns from '../UserButton/UserFormBtn';
+import UserRadioBtns from '../UserRadioBtns/UserFormRadio';
+
+// STYLES
+import * as S from './AdminEle';
+
+// import { BASE_URL } from '../../../config';
+
+function Admin({ form, modalClose }) {
+  const { Kakao } = window;
+  const [radio, setRadio] = useState('');
+  const [inputValues, setInputValues] = useState({
+    name: '',
+    email: '',
+    password: '',
+  });
+
+  const handleSubmit = e => {
+    e.preventDefault();
+  };
+
+  const getInputValue = e => {
+    const { name, value } = e.target;
+
+    setInputValues({ ...inputValues, [name]: value });
+  };
+
+  const getRadioValue = e => {
+    setRadio(e.target.value);
+  };
+
+  const handleClickButton = e => {
+    const { name } = e.target;
+    console.log(`name`, name);
+
+    if (name === 'email') {
+      fetchLogin();
+      setInputValues({ name: '', email: '', password: '' });
+    } else {
+      // localStorage.removeItem('token');
+    }
+  };
+
+  const fetchLogin = () => {
+    fetch(`http://13.125.45.70/master/signin`, {
```

Comment:
fetch 첫 번째 인자 url은 항상 src/config.js에 분리하여 관리해주세요.
```suggestion
    fetch(`${API}/master/signin`, {
```

Reply:
> fetch 첫 번째 인자 url은 항상 src/config.js에 분리하여 관리해주세요.

알겠습니당 !